README.md

```
@@ -1,21 +1,71 @@
-# reto-winter-2025-ithaka-frontend
+# Reto Winter 2025 - Ithaka Frontend
 
-## Getting Started
+## Proposito  
+Este proyecto tiene como objetivo desarrollar un componente de asistente conversacional con IA, diseñado para integrarse 
+fácilmente en la página web de Ithaka. El componente permitirá a los usuarios interactuar con un chatbot inteligente, 
+ofreciendo soporte, guía o acompañamiento en tiempo real, adaptado a las necesidades de cada proyecto o emprendimiento.  
+Entre las funcionalidades principales están la respuesta a preguntas frecuentes y el acompañamiento y asesoriamiento en la 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)